lib/decidim/direct_verifications/version.rb

```diff
@@ -3,8 +3,8 @@
 module Decidim
   # This holds the decidim-direct_verifications version.
   module DirectVerifications
-    VERSION = "1.2.1"
-    DECIDIM_VERSION = "0.26.1"
-    MIN_DECIDIM_VERSION = ">= 0.25.0"
+    VERSION = "1.3.0"
+    DECIDIM_VERSION = "0.27.2"
+    MIN_DECIDIM_VERSION = ">= 0.27.2"
```

Comment:
Please, change the `MIN_DECIDIM_VERSION` to `[">= 0.27.0", "< 0.28"].freeze` because we can't guarantee it will keep working on 0.28. To make it more readable, you could also change the name of the constant to `COMPAT_DECIDIM_VERSION`